### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by fgrehm**
### TL;DR

This is a 8+yr old gem, it's time for a new major release and some breaking changes. We also need to give some love to the UI since it hasn't changed at all since the [very first release](http://fabiorehm.com/blog/2013/01/22/announcing-letter_opener_web-1-0/).

### How to test?

Set the gem to the new pre-release version:

```ruby
gem 'letter_opener_web', '2.0.0.pre.beta'
```

Then `bundle install`, restart your app and head over to the app's inbox (ex: http://localhost:3000/inbox) and you'll get the new release :tada: 

### New look

image

### TODO

- [x] Upgrade boostrap, jquery, etc
- [x] Show email summary on sidebar
- [x] Update `v1-stable` branch with what's on master
- [x] Merge in these PRs
  - [x] https://github.com/fgrehm/letter_opener_web/pull/106
  - [x] https://github.com/fgrehm/letter_opener_web/pull/108
  - [x] https://github.com/fgrehm/letter_opener_web/pull/111
  - [x] https://github.com/fgrehm/letter_opener_web/pull/112
  - [x] https://github.com/fgrehm/letter_opener_web/pull/69
  - [x] https://github.com/fgrehm/letter_opener_web/pull/114
- [x] Update demo app
- [x] [CI with github actions, including auto release](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-ruby)
- [ ] [Fix integration with rails api apps for good, or document how to make things work if we can't handle all of it in the gem](https://github.com/fgrehm/letter_opener_web/pull/69#issuecomment-955263951)
- [x] [Bring back delete button for individual msgs](https://github.com/fgrehm/letter_opener_web/pull/111#issuecomment-953862267)
- [x] Close stale issues
- [x] Update changelog

### CODE REVIEW

**Comment by fgrehm** [comment]
Thanks @tricknotes! I think we're very close to a final release, there's just [one thing @dee-see found](https://github.com/fgrehm/letter_opener_web/pull/111#issuecomment-953862267) that I'd like to do before that :crossed_fingers:

**Comment by ihatov08** [comment]
@fgrehm 
I used new revision in my sample application(https://github.com/ihatov08/letter_opener_web_rails_api),
csrf error didn't occur.(and flash error didn't occur)
It looks good!

**Comment by fgrehm** [comment]
Thanks @ihatov08! I'm going to go ahead and cut the final release now :tada:
